src/redux/store.js

```diff
@@ -10,7 +10,7 @@ import mainPageReducer from './mainPage/mainPageSlice';
 const persistConfig = {
   key: 'root',
   storage,
-  whitelist: ['auth', 'mainPage'],
+  whitelist: ['auth', 'mainPage', 'doctor'],
```

Comment:
The doctor reducer should not be whitelisted here. Whitelist is only for the state which should remain after refreshing the page.